src/App.css

```diff
@@ -1,3 +1,26 @@
-.App {
+.body {
```

Comment:
Why use a class of body. Why not just body? Use element selectors wherever they make sense. Class selectors are only when you need more specificity.